### PR TITLE
Remove sentry

### PR DESCRIPTION
**Pull request by augustuswm**
* Removes sentry configuration from the server and client
* Removes sentry packages

### CODE REVIEW

File: app/entry.client.tsx [hunk 1]
```diff
@@ -6,28 +6,10 @@
...
-        routingInstrumentation: Sentry.remixRouterInstrumentation(
-          useEffect,
-          useLocation,
-          useMatches,
-        ),
-      }),
-    ],
-  })
-}
-
```
Comment (david-crespo) [inline app/entry.client.tsx:30]: I bet we can delete this file altogether now. If you don't have one it should use the built-in default.

File: app/entry.server.tsx [hunk 2]
```diff
@@ -9,18 +9,9 @@
...
 import { renderToPipeableStream } from 'react-dom/server'
 
-if (typeof process.env.SENTRY_DSN === 'string') {
-  Sentry.init({
-    dsn: process.env.SENTRY_DSN,
-    tracesSampleRate: 0.1,
-    enabled: process.env.NODE_ENV === 'production',
-  })
-}
-
```
Comment (david-crespo) [inline app/entry.server.tsx:23]: Actually... I bet we can delete this file too. It's only in the other apps that we're doing the cache header thing that keeps this file around.

**Review by david-crespo — APPROVED** [review]
Looks great. Try deleting the entry files.

**Comment by benjaminleonard** [comment]
Sorry I took so long to get to this. Removed in the Remix v2 upgrade.

**Pull request closed without merging**